server/routes/overview/mappers/functionalSkillsMapper.test.ts

```diff
@@ -11,7 +11,7 @@ describe('functionalSkillsMapper', () => {
       {
         prn: prisonNumber,
         establishmentId: 'MDI',
-        establishmentName: 'MOORLAND (HMP & YOI)',
+        establishmentName: 'Moorland (HMP & YOI)',
```

Comment:
No, this is not right. These values are ALWAYS uppercase in the data we get back from Curious (which is why we do a prison name lookup)